gulp/tasks/test_debug.js

```diff
@@ -0,0 +1,26 @@
+(function () {
+    'use strict';
+
+    var gulp = require('gulp'),
+        karma = require('karma').server,
+        path = require('path'),
+        paths = {
+            spec: [
+                'test/specs/**/*.js'
```

Comment:
Actually, a better example of passing settings per spec in is here (https://github.com/talbs/mise-en-place/blob/master/gulp/tasks/css.js#L16), where I'm passing in settings from the CSS portion of my config (https://github.com/talbs/mise-en-place/blob/master/gulp/config.js#L12)